packages/std/src/math/decimal.rs

```diff
@@ -696,26 +704,77 @@ mod tests {
         // Decimals
         assert_eq!(Decimal::percent(125).to_string(), "1.25");
         assert_eq!(Decimal::percent(42638).to_string(), "426.38");
-        assert_eq!(Decimal::percent(1).to_string(), "0.01");
+        //assert_eq!(Decimal::percent(1).to_string(), "0.01");
```

Comment:
What happened here? Want to avoid the duplicate test?

```suggestion
        assert_eq!(Decimal::percent(3).to_string(), "0.03");
```